pkg/ddl/index.go

```diff
@@ -2449,9 +2450,22 @@ func checkDuplicateForUniqueIndex(ctx context.Context, t table.Table, reorgInfo
 		if indexInfo.Unique {
 			ctx := tidblogutil.WithCategory(ctx, "ddl-ingest")
 			if bc == nil {
+				var (
+					cfg *local.BackendConfig
+					bd  *local.Backend
```

Comment:
seems `bd` can be shared in this for loop. Create it lazily and after the loop close if not nil.